CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 ### Other changes
 
 * Fixed a retain cycle in CarPlay implementation of a navigation map view that prevented `NavigationMapView` instances from being deallocated after CarPlay is stopped. ([#3552](https://github.com/mapbox/mapbox-navigation-ios/pull/3552))
+* Fixed a regression when no congestion level information found, the current active route leg shows `NavigationMapView.routeCasingColor` instead of `NavigationMapView.trafficUnknownColor`. ([#3577](https://github.com/mapbox/mapbox-navigation-ios/pull/3577))
```

Comment:
```suggestion
* Fixed an issue where the entire route line was colored as `NavigationMapView.routeCasingColor` instead of `NavigationMapView.trafficUnknownColor` when traffic congestion data was missing. ([#3577](https://github.com/mapbox/mapbox-navigation-ios/pull/3577))
```